includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -248,29 +252,57 @@ private function render_module_block( $block, $course_id, $outline_attributes )
 
 		$animated = empty( $outline_attributes['animationsEnabled'] ) ? '' : 'animated';
 
+		$class_name = Sensei_Block_Helpers::block_class_with_default_style( $block['attributes'] );
+
+		$is_default_style = false !== strpos( $class_name, 'is-style-default' );
+		$is_minimal_style = false !== strpos( $class_name, 'is-style-minimal' );
+
+		$header_css = Sensei_Block_Helpers::build_styles(
+			$block,
+			[
+				'mainColor' => $is_default_style ? 'background-color' : null,
+			]
+		);
+
+		$style_header = '';
+
+		if ( $is_minimal_style ) {
+
+			$header_border_css = Sensei_Block_Helpers::build_styles(
+				$block,
+				[
+					'mainColor' => 'background-color',
+				]
+			);
+
+			$style_header = '<div ' . Sensei_Block_Helpers::render_style_attributes( 'wp-block-sensei-lms-course-outline-module__name__minimal-border', $header_border_css ) . '></div>';
```

Comment:
An additional element is used for the Minimal style module header border, so that it can use named theme colors via background-color, because classes for border color are usually not provided by themes.